src/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable */
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink/PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleList: React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <tr
+            data-cy="person"
+            key={person.slug}
+            className={
+              person.slug === personSlug ? 'has-background-warning' : ''
+            }
+          >
+            <td>
+              <PersonLink person={person} />
+            </td>
+
+            <td>{person.sex}</td>
+            <td>{person.born}</td>
+            <td>{person.died}</td>
+            <td>
+              {people.find(persName => person.motherName === persName.name) ? (
+                <PersonLink
+                  person={people.find(p => p.name === person.motherName)!}
```

Comment:
One-letter naming of the argument is not a good idea. Please make it more readability.